qt/scientific_interfaces/ISISReflectometry/GUI/Runs/RunsPresenter.cpp

```diff
@@ -424,35 +454,17 @@ void RunsPresenter::transfer(const std::set<int> &rowsToTransfer,
  */
 void RunsPresenter::updateWidgetEnabledState() const {
   // Update the menus
-  // TODO: reinstate isProcessing when implemented
-  m_view->updateMenuEnabledState(false /*isProcessing()*/);
+  m_view->updateMenuEnabledState(isProcessing());
 
   // Update components
-  m_view->setTransferButtonEnabled(/*!isProcessing() &&*/ !isAutoreducing());
-  m_view->setInstrumentComboEnabled(/*!isProcessing() &&*/ !isAutoreducing());
-  m_view->setAutoreducePauseButtonEnabled(isAutoreducing());
+  m_view->setInstrumentComboEnabled(!isProcessing() && !isAutoreducing());
```

Comment:
`const bool shouldEnable = !isProcessing() && !isAutoreducing();` and then pass `shouldEnable` into the methods to avoid repeated calls to `isProcessing` and `isAutoreducing`?

Reply:
I'd rather avoid a vague name like `shouldEnable` because different widgets have different conditions under which they should be enabled. I could call it `notProcessingOrAutoreducing` but that probably wouldn't help with readability!